### DISCUSSION TITLE
filter selection for DenoiseImage

### DISCUSSION THREAD

**Discussion by md06451** [ANTsX/ANTs#1802] [Q&A] [answered]
Dear all, 
I aim to use DenoiseImage to denoise the fast spin echo t1w scans (11 slices). 
I read the paper based on which the algorithm is based https://doi.org/10.1002/jmri.22003 from Manjon. While they direct the reader on the selection of Rician filter for T2w, they do not write what we should choose for t1w scans. I see that the default settings in DenoiseImage is Gaussian and Gaussian was also used in an application of the Longitudinal Mapping of Cortical Thickness pipeline of with T1w (https://pmc.ncbi.nlm.nih.gov/articles/PMC10204115/; https://github.com/ANTsX/ANTs/blob/master/Scripts/antsLongitudinalCorticalThickness.sh). However, other studies (Coupe, 2011; 10.1016/j.neuroimage.2010.09.018) used Rician for t1w. What is your advice? which filter should I use? Thank you in advance.

**Comment by ntustison** [comment, marked as answer]
In all of our early exploration, we tended to prefer the Gaussian results, in particular for T1-w images which is why that's the default.  That's what I would recommend to you although, ultimately you should do the comparisons yourself and see which one best suits your purposes.

**Comment by md06451** [comment]
Hi all, I have another question on the output of denoise in ants. I saw that it is possible to extract the estimated noise map (-o [correctedImage,<noiseImage>]). From the paper of Manjón, I understand that the noise/variance is calculated based on the Euclidean distance of two noisy patches in the Diff volume. The Diff volume is the difference between the original noisy scan and a low pass-filtered volume (https://onlinelibrary.wiley.com/doi/10.1002/jmri.22003).  Am I correct in interpreting that we can safely compare and pool estimated noise maps from different scans/studies (No normalization is needed)?

**Reply by ntustison** [reply]
I would direct this question to the original author.
